/*
抛出异常

1. 系统抛出的
2. 自己抛出的

 */
void distanceTo() => throw UnimplementedError("abcd");
void test() => throw Error();
void main() {
  //distanceTo();
  //test();
  //throw FormatException('Expected at least 1 section');
 // throw Exception('异常的根');
  //throw '溢出了.';

}